src/Modules/Engine.cpp

```diff
@@ -648,6 +652,13 @@ void _Host_RunFrame_Render_Detour();
 void (*_Host_RunFrame_Render)();
 static Hook _Host_RunFrame_Render_Hook(&_Host_RunFrame_Render_Detour);
 void _Host_RunFrame_Render_Detour() {
+	g_total_frames++;
+
+	if (now != time(NULL)) {
```

Comment:
put `time(NULL)` in a variable before here and use that, else you have a race condition with the line below